http-server/apache2-web-server/README.md

```diff
@@ -120,13 +139,17 @@ Or:
 $ w3m http://localhost:8080
 ```
 
-And access the `index.html` file being served from the `web/htdocs/` location.
+<br>
 
+Remember, you must open the web page on the same host thats running the instance.
```

Comment:
Remember, you must open the web page on the same host **that is** running the instance